packages/apps/desktop-viewer-test/.env

```diff
@@ -1,8 +1,9 @@
 # ---- Configuration ----
-ITWIN_VIEWER_SCOPE ="email rbac-user:external-client projectwise-share product-settings-service context-registry-service:read-only imodelhub organization profile openid"
+ITWIN_VIEWER_SCOPE ="email organization profile openid itwinjs"
 ITWIN_VIEWER_CLIENT_ID=""
 ITWIN_VIEWER_REDIRECT_URI=""
 ITWIN_VIEWER_SNAPSHOT="./samples/house_model.bim"
+ITWIN_VIEWER_ISSUER_URL=""
```

Comment:
I'd set the default Issuer to be ims.bentley.com now since you're using that scope by default. Then just provide the option to review back...